index.js

```diff
@@ -499,6 +541,21 @@ async function findAllSchemasInfo(schemaBasePath, options = {}) {
     )).sort((a, b) => semver.gt(a.version, b.version));
 }
 
+/**
+ * Synchronous version of findAllSchemasInfo
+ * @param {string} schemaBasePath
+ * @param {Object} options
+ * @return {Object[]}
+ */
+function findAllSchemasInfoSync(schemaBasePath, options = {}) {
+    _.defaults(options, defaultOptions);
+
+    const schemaPaths = findSchemaPathsSync(schemaBasePath, options);
+    // Map each schema path to a schema info object, including the schema itself.
+    return schemaPaths.map(schemaPath => schemaPathToInfoSync(schemaPath, options))
```

Comment:
Let's make a common function that takes the filtered schemaPaths so we can DRY up e.g. this sort (that I changed in my PR)

Reply:
Not really relevant anymore